lib/active_record_doctor/detectors/missing_unique_indexes.rb

```diff
@@ -60,6 +62,20 @@ def supported_validator?(validator)
           validator.options.fetch(:case_sensitive, true)
       end
 
+      def resolve_attributes(model, attributes)
+        attributes.map do |attribute|
+          reflection = model.reflect_on_association(attribute)
+
+          if reflection.nil?
+            attribute
+          elsif reflection.polymorphic?
+            [reflection.foreign_key, reflection.foreign_type]
```

Comment:
I think it'd make more sense to have the type appear first, followed by ID, as that index could also participate in fetching related models (e.g. "fetch all comments left on articles", which could use the first column in an index on `(type, id)`).

Reply:
Yes, the order doesn't matter for uniqueness but it matters for **other** queries. If we're recommending creating an index then let's recommend something the option with broader practical applications so that the user won't need to create another index on the same columns but in reverse order.

Reply:
@gregnavis Ahh, sure 🤦 Updated.